contracts/ArrowVestingBase.sol

```diff
@@ -5,8 +5,10 @@ import "@openzeppelin/contracts-upgradeable/finance/VestingWalletUpgradeable.sol
 
 /** 
     @title Base implementation of vesting contract that will be cloned into individual vesting wallets pointing at specific beneficiary addresses.
+    @dev Although the contract inherits an upgradeable interface, the contract itself is not intended to be upgradeable
```

Comment:
It is intended to be clone-able though, which is why we need to use an upgradable contract, so that the factory contract can successfully clone it.